### PR TITLE
docs: update to twilio cli commands

### PR DESCRIPTION
**Pull request by jonedavis**
-remove console and ngrok set up
- added twilio cli steps to provision number and webhook proxy to number

### CODE REVIEW

**Comment by jonedavis** [comment]
@dkundel can you 👀? Ready to move to code-exchange.

File: README.md [hunk 1]
```diff
@@ -75,19 +75,54 @@ cd sample-pay-service
 
 ```bash
 npm install
-npm install -g ngrok
 ```
 
-3. Set your environment variables
+3. Install Twilio CLI
 
+The Twilio CLI allows you to manage your Twilio resources from your terminal or command prompt. You can use the Twilio CLI below to a provision phone number, configure the webhook, and run ngrok to expose your `localhost` to a public domain so the Twilio webhook can reach the expected endpoint. Alternatively, You can do this in the Twilio Console and by installing `ngrok` separately.
```
Comment (dkundel) [inline README.md]: I think we should either just link to the quickstart here or only show how to install the Twilio CLI using `npm`. Everything else seems confusing. Plus for this use case you'll have `npm` anyways installed

File: README.md [hunk 2]
```diff
@@ -75,19 +75,54 @@ cd sample-pay-service
...
+> See [Twilio CLI Quickstart](https://www.twilio.com/docs/twilio-cli/quickstart) for installation guide for Linux and to learn how to use the Twilio CLI.
+
+4. Log in to Twilio CLI
+```bash
+twilio login
+```
+
+See [Twilio Account Settings](#twilio-account-settings) to locate the necessary environment variables.
+
+5. List phone numbers available for purchase by area code
```
Comment (dkundel) [inline README.md]: There are better commands here:
`twilio phone-numbers:list` and `twilio phone-numbers:buy:...`

File: README.md [hunk 3]
```diff
@@ -99,23 +134,20 @@ Alternatively, you can use this command to start the server in development mode.
...
+10. Navigate to [http://localhost:3000](http://localhost:3000) to see some sample credit card details to test the payment.
 
-8. You can also navigate to [http://localhost:3000/config](http://localhost:3000/config) to override the default payment details.
+11. You can also navigate to [http://localhost:3000/config](http://localhost:3000/config) to override the default payment details.
 
-9. That's it! Now call the Twilio phone number you configured and follow the instructions to complete the payment.
+12. That's it! Now call the Twilio phone number you configured and follow the instructions to complete the payment.
 
-10. You can see if the payment was charged on your Stripe dashboard. Take a look at [this](https://www.twilio.com/docs/voice/tutorials/how-capture-your-first-payment-using-pay#test-your-application) for more details.
+13. You can see if the payment was charged on your Stripe dashboard. Take a look at [this](https://www.twilio.com/docs/voice/tutorials/how-capture-your-first-payment-using-pay#test-your-application) for more details.
```
Comment (dkundel) [inline README.md]: The `[this]` link should be more descriptive for accessibility and SEO. Something like `[Take a look at this tutorial]`

**Review by dkundel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jonedavis) [reply README.md]: is `twilio phone-numbers:list` only to show the numbers you own? Looking to list all numbers in area code to purchase. The phone-numbers:buy is new and cool to me. Probably should put that front and center in cli docs.

File: README.md (hunk 3 above)
Comment (jonedavis) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (jonedavis) [reply README.md]: Oh i see it's consolidated

`twilio phone-numbers:buy:local --country-code US --area-code="208"`

File: README.md (hunk 1 above)
Comment (jonedavis) [reply README.md]: 👍

**Review by dkundel — APPROVED** [review]

**Pull request merged**